### PR TITLE
adds windows option to the whole tutorial

### PR DESCRIPTION
**Pull request by ercchy**

### CODE REVIEW

File: 1-installation/index.md [hunk 1]
```diff
@@ -3,27 +3,40 @@ layout: tutorial
...
+You'll create your website with a framework called [Jekyll](http://jekyllrb.com). Despite the unusual name, Jekyll is a nice tool you can use to create simple, yet flexible websites that can grow with you as you learn more about various web technologies.
 
 Before we can get started, let's take a few minutes to get familiar with an app that allows you to command your computer. We'll also install a few tools on your computer that will make your work easier.
 
-## Get to know the Terminal
 
-<img src="{{ "/images/terminal-icon-mac.png" | prepend: site.baseurl }}" class="text-icon"/> First, find and open an app called Terminal on your computer. It's usually in the `/Applications/Utilities` folder or you can use Spotlight Search to find it. 
+## Get to know the <span class="os-switch"><span class="osx">Terminal</span><span class="wins">Command Prompt</span></span>
+<img src="{{ "/images/terminal-icon-mac.png" | prepend: site.baseurl }}" class="icon"/> First, find and open an app called Terminal on your computer.
```
Comment (ialja) [inline 1-installation/index.md]: img class should be text-icon

File: 1-installation/index.md [hunk 2]
```diff
@@ -35,82 +48,158 @@ Hint: don't worry if you make a typo when writing commands into the <span class=
...
+<span class="osx">
+<div class="highlighter-rouge"><pre class="highlight"><code>$ rm -r cat-folder
+</code></pre></div>
+</span>
+<span class="wins">
+<div class="highlighter-rouge"><pre class="highlight"><code>> rmdir /s cat-folder
+</code></pre></div>
+</span>
+</div>
+<code class="highlighter-rouge"><span class="os-switch"><span class="osx">rm</span><span class="wins">rmdir</span></span></code> stands for "**r**e**m**ove". Be careful when using this command, there's no undo, so make sure you're only deleting files and folder you don't need anymore.
```
Comment (ialja) [inline 1-installation/index.md]: Include the explanation in the os switcher (on Win it should be: rmdir stands for “remove directory")

File: 1-installation/index.md [hunk 3]
```diff
@@ -142,17 +231,32 @@ If you get a line that starts with `ruby`, followed by version number that is `1
...
+<div class="highlighter-rouge"><pre class="highlight"><code>$ sudo gem install github-pages bundler
+</code></pre></div>
+
+</div>
+<div class="wins">
+
+Sometimes things don't go smoothly and you can get pretty weird errors when you try to install gems. One of the common ones is when you run the above command you get something like this:
+<div class="highlighter-rouge"><pre class="highlight"><code> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed</code></pre></div>
+
+Don't worry. In a typical day of a programmer there is more then one such moment and it's pretty normal to Google for solutions. So if this happens to you, we found a <a here="http://guides.rubygems.org/ssl-certificate-update/" target="_blank">page</a> where you can follow the steps and resolve this issue.
```
Comment (ialja) [inline 1-installation/index.md]: Link not working (having "href" instead of "here" should fix it :) )

File: 1-installation/index.md [hunk 4]
```diff
@@ -35,82 +48,158 @@ Hint: don't worry if you make a typo when writing commands into the <span class=
...
+16.10.2015  11:06  Downloads
+03.05.2015  17:17  Favorites
+...
+</code></pre></div>
+</span>
+</div>
 
 Now, let's move into the `Desktop` folder:
 
 	$ cd Desktop
```
Comment (ialja) [inline 1-installation/index.md]: I think lines like this should also get a Windows version, so it's not confusing to see $ and > changing throughout the tutorial.

Comment (ercchy) [reply 1-installation/index.md]: Fixeth, please review.

**Comment by ialja** [comment]
Such awesome job on #12 

There is still a $ jekyll serve in part two and some Terminals to turn into cmds, but this can be updated later.

**Pull request merged**